### DISCUSSION TITLE
Total newb here - which version?

### DISCUSSION THREAD

**Discussion by ctankep23** [bluenviron/mediamtx#1006] [Questions]
Like the title says, I am a humble simple person who cant seem to figure out which of the binaries I am supposed to install. I am gonna use an old Macbook Pro with dual core intel i7 cpu to act as the rtsp server. Please dont make me feel any more dumb than I already am, but could someone please point me in the right direction?

**Comment by aler9** [comment]
Hello, macOS is based on Darwin OS, therefore the right binary to download is the darwin_amd64 one.

[a comment by an automated account is not shown]
